agent/agent/main.go

```diff
@@ -38,7 +38,7 @@ func main() {
 		fmt.Println("Must specify rootUrl.")
 		return
 	}
-	svcInfo, err := agent.Run(*rootURL)
+	svcInfo, err := agent.Run(*rootURL, false)
```

Comment:
It saddens me that Go doesn't seem to have named parameters. I need to go all the way to the function definition to see what that boolean here is for. This here would be so much nicer and self-documenting:

```
svcInfo, err := agent.Run(*rootURL, testMode=false)
```

There's no decent equivalent for this in Go?

Do you really have to write:

```
svcInfo, err := agent.Run(*rootURL, false)  // flag indicates: no test mode
```

???